website/docs/reference/resource-properties/config.md

```diff
@@ -109,6 +110,23 @@ version: 2
 
 </TabItem>
 
+<TabItem value="unit tests">
+
```

Comment:
I think unit tests came in 1.8 -- should we have a 'Available from in 1.8 and higher'.. sentence?

Reply:
going to add it to the page